src/pages/be-square/components/_advantages.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { CardContainer } from '../static/style/_advantages'
+import { Title } from '../static/style/_common'
+import advantages from '../static/content/_advantages'
+import Card from './_card'
+import { Container } from 'components/containers'
+
+const Advantages = () => {
+    const style = {
+        card_wrapper: {
+            width: ['auto'],
+            max_width: ['auto', '290px'],
+            height: ['244px', '180px'],
+            padding: ['32px 0 32px 32px'],
```

Comment:
Is it intentional for the right padding to be 0? As the words end up touching the right side border of the card for some screen widths.

Reply:
Good catch. Will fix that 👍🏻